src/main/java/org/broadinstitute/dsm/db/KitType.java

```diff
@@ -105,7 +102,7 @@ public KitType(int kitId, String name, String displayName, boolean manualSentTra
             try (PreparedStatement stmt = conn.prepareStatement(query)) {
                 stmt.setString(1, realm);
                 try (ResultSet rs = stmt.executeQuery()) {
-                    while (rs.next()) {
+                    if (rs.next()) {
```

Comment:
Is it an error if there are > 1 rows?  If so, that should be checked here and thrown.